pkg/controllers/clusterresource/keymanagementprovider_controller.go

```diff
@@ -52,10 +50,10 @@ type KeyManagementProviderReconciler struct {
 func (r *KeyManagementProviderReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	logger := logrus.WithContext(ctx)
 
-	var resource = req.NamespacedName.String()
+	var resource = req.Name
 	var keyManagementProvider configv1beta1.KeyManagementProvider
 
-	logger.Infof("reconciling key management provider '%v'", resource)
+	logger.Infof("reconciling cluster key management provider '%v'", resource)
 
 	if err := r.Get(ctx, req.NamespacedName, &keyManagementProvider); err != nil {
```

Comment:
for cluster resource, do we use the namespacedName? or is this automatically the same as Name?

Reply:
NamespacedName would prefix a '/', like `/test`, but Name is just `test`. As for cluster resource, `namespacedName` also works since and verifier/store do the same previously.